types/simple-icons/simple-icons-tests.ts

```diff
@@ -30,3 +31,12 @@ for (const slug in simpleIcons) {
 
 // direct import
 simpleicons; // $ExpectType SimpleIcon
+
+const simpleIcon: SimpleIcon = {
+    title: 'Simple Icon',
+    slug: 'simpleicons',
+    svg: '<svg></svg>',
+    path: 'path/to/icon.svg',
+    source: 'https://simpleicons.org/',
+    hex: '#000000',
```

Comment:
hex values are without a leading `#`

```suggestion
    hex: '000000',
```